### DISCUSSION TITLE
Kurbağa Yollarını Saymak

### DISCUSSION THREAD

**Discussion by FatihOnder81** [sonsuzus/sonsuzus#404] [Soru Cevap]
Sorun
Koordinat düzleminin başlangıç ​​noktasında bir kurbağa yer alıyor. Tam olarak
ikincisi, kurbağa ya yukarı doğru hareket edebilir
birim, sağa hareket et
Birlik olun ya da hareketsiz kalın. Başka bir deyişle pozisyondan
kurbağa harcayabilir
ikinci olarak şuraya taşınalım:

Sonrasında
Birkaç saniye sonra kurbağayı gören bir köylü, kurbağanın kenar uzunluğundaki bir karenin üzerinde veya içinde yattığını bildirir.
 koordinatlarla
,
,
,
. Bu karenin üzerinde veya içinde koordinatları tam sayı olan kaç noktanın tam olarak sonra kurbağanın konumu olabileceğini hesaplayın
saniye

Giriş Formatı:

Girişin ilk ve tek satırı boşlukla ayrılmış dört tamsayı içerir: 
, 
, 
, Ve 
.

Çıkış biçimi:

Kurbağanın konumu olabilecek tamsayı koordinatlı noktaların sayısını yazdırın 
 saniye.

Kısıtlamalar:

Özel Çağrı için Beklenen Çıktı Özelliğinin bu yarışmada desteklenmediğini unutmayın.

**Comment by Ahmet2938** [comment]
def count_frog_positions(t, x1, y1, x2, y2):
    count = 0
    for x in range(t + 1):
        y = t - x        
        if x1 <= x <= x2 and y1 <= y <= y2:
            count += 1
    return count
t, x1, y1, x2, y2 = map(int, input("Saniye ve kare köşelerini girin: ").split())
result = count_frog_positions(t, x1, y1, x2, y2)
print(result)
